src/microfw.sh

```diff
@@ -142,6 +148,7 @@ function generate_setup() {
 
 function compile() {
     mkdir -p $VAR_DIR
+    mkdir -p $SHM_DIR
```

Comment:
```suggestion
    mkdir -p $RUN_DIR
```

Reply:
If systemd will create the directory `/run/microfw` we could get rid of the whole line.